BlueEastCode/bluerain-ui-interfaces
# Tab and Tabs interface

## What I did

- created new tabs and tab Interfaces

## How to test

Is this testable with jest or storyshots?

Are any of the existing tests are failing ?

Does this need an update to the documentation?

Does it have breaking changes ?

If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
